backend/routes/links.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../config/database");
+
+const Link = db.Link;
+
+router.get("/", (req, res, next) => {
+    Link.findAll({
+        limit: 2, 
+        order: '"id" ASC'
+    }).then(data=> res.send(data));
+});
+router.get("/:id", (req, res, next) => {
+    Link .findAll({ where: {
+        id: req.params.id
+      }}).then(data=>res.send(data))
+});
+
+router.post("/", (req, res, next) => {  
+    Link.create(req.body).then(data=> res.send(data));
+ });
+
+
+router.get("/upClickCount/:id",(req,res,next) => {
```

Comment:
What are upclicks for? Are we counting how many times a user clicks a link on a published story?

Reply:
removed this